js/apps/system/_admin/aardvark/APP/react/src/views/collections/indices/listIndices/CollectionIndicesTable.tsx

```diff
@@ -54,6 +47,50 @@ const TABLE_COLUMNS = [
       filterType: "multi-select"
     }
   }),
+  columnHelper.accessor(
+    row => {
+      return row.fields
+        .map(field => {
+          if (typeof field === "string") return field;
+          return field.name;
+        })
+        .join(", ");
+    },
+    {
+      header: "Fields",
+      id: "fields",
+      meta: {
+        filterType: "text"
+      }
+    }
+  ),
+  columnHelper.accessor("unique", {
+    header: "Unique",
+    id: "unique",
+    cell: info => {
+      return `${info.cell.getValue()}`;
+    },
+    meta: {
+      filterType: "single-select",
+    }
+  }),
+  columnHelper.accessor(
+    row => {
+      if ((row as any).selectivityEstimate) {
+        const selectivityEstimate = (row as any).selectivityEstimate;
+        // percentage rounded to 2 decimal places
+        return `${(selectivityEstimate * 100).toFixed(2)}%`;
```

Comment:
Out of curiosity, do we want to round it?

Reply:
I've kept it the same as 3.10/3.11, hence the rounding